frontend/src/components/Login/Login.js

```diff
@@ -36,19 +38,28 @@ const Login = () => {
 
   const loginHandler = async (e) => {
     e.preventDefault()
-    setFormErrors(validateForm(user))
+    const err = validateForm(user)
+    setFormErrors(err)
     try {
-      setError("")
-      const res = await axios.post(`${apiUrl}/users/login`, user)
-      if (res.data.user && res.data.token) {
-        localStorage.setItem("user", JSON.stringify(res.data.user))
-        localStorage.setItem("token", res.data.token)
-        navigate("/profile")
-      } else {
-        throw new Error("Invalid login")
+      if (Object.keys(err).length < 1) {
+        let res = await axios.post(`${apiUrl}/users/login`, user)
+        if (res.data.user && res.data.token) {
+          localStorage.setItem("user", JSON.stringify(res.data.user))
+          localStorage.setItem("token", res.data.token)
+          navigate("/profile")
+        } else {
+          return error
+        }
       }
     } catch (err) {
-      if (err.response) {
+      let res = err?.response?.data
+      if (res.message === "No user exists! Please Register") {
```

Comment:
toast.error(res.message) 
Can't we write this?

Reply:
yes, updated